test/test_sparse.py

```diff
@@ -3427,6 +3427,21 @@ def test_cuda_sparse_cpu_dense_add(self):
         with self.assertRaisesRegex(RuntimeError, "add: expected 'self' to be a CUDA tensor, but got a CPU tensor"):
             x + sparse_y
 
+    def test_cpu_sparse_dense_mul(self):
```

Comment:
Is the error present for the CUDA device?

Reply:
no, they raise a NotImplementedError